app/src/main/java/com/udacity/popularmovies/DetailActivity.java

```diff
@@ -12,6 +12,7 @@
 import com.udacity.popularmovies.utilities.JsonUtils;
 
 public class DetailActivity extends AppCompatActivity {
+    private static final String MOVIE = "movie";
```

Comment:
[nit] Since this constant is used twice by different Activities, would be best to put into a separate constants file for both to reference

Reply:
It's actually considered bad practice in Java to have a separate constants file (and in general for OOP), usually want to put into interface that's shared among the classes. Would need to add to `AppCompatActivity` interface since Java doesn't allow for multiple inheritance.

Resource: https://stackoverflow.com/questions/3866190/java-constants-file